build/LegacyPackages/14.0/Microsoft.Build.Engine/Microsoft.Build.Engine.nuspec

```diff
@@ -0,0 +1,37 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata minClientVersion="2.8.1">
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>$id$</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=624683</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <description>This package contains the $id$ assembly which contains the core MSBuild runtime.</description>
```

Comment:
>the core MSBuild runtime [](start = 72, length = 24)

This package actually contains the legacy compatibility shim for the engine, which is in `Microsoft.Build`. It's also obsolete.